runtime/kusama/src/lib.rs

```diff
@@ -1036,6 +1043,24 @@ construct_runtime! {
 	}
 }
 
+impl pallet_babe::migrations::HasPalletPrefix for Runtime {
+	fn pallet_prefix() -> &'static str {
+		"BabeApi"
+	}
+}
```

Comment:
I ran `wasm-nm -e target/debug/wbuild/kusama-runtime/kusama_runtime.compact.wasm` and it appears that `BabeApi` is the correct prefix at the moment:
```
e BabeApi_configuration
e BabeApi_current_epoch_start
e BabeApi_current_epoch
e BabeApi_next_epoch
e BabeApi_generate_key_ownership_proof
e BabeApi_submit_report_equivocation_unsigned_extrinsic
```

Reply:
No, that is not the correct prefix. This is the name of the runtime api, not the prefix of the storage. The prefix currently is `Babe`, as you see here https://github.com/paritytech/substrate/blob/master/frame/babe/src/lib.rs#L161